pkg/infoschema/infoschema_v2.go

```diff
@@ -455,6 +455,28 @@ func (is *infoschemaV2) EvictTable(schema, tbl string) {
 	is.tableCache.Remove(tableCacheKey{itm.tableID, itm.schemaVersion})
 }
 
+type helper struct {
+	end           tableItem
+	schemaVersion int64
+	ok            bool
+	itm           tableItem
+}
+
+func (h *helper) iter(item tableItem) bool {
```

Comment:
```suggestion
func (h *helper) onItem(item tableItem) bool {
```

Reply:
i don't like the name in `btree` pkg, `iter` mostly means something with open/next/close